internal/apis/certmanager/types_issuer.go

```diff
@@ -292,6 +292,13 @@ type CAIssuer struct {
 	// certificate will be issued with no OCSP servers set. For example, an
 	// OCSP server URL could be "http://ocsp.int-x3.letsencrypt.org".
 	OCSPServers []string
+
+	// The CA Issuers server list is an X.509 v3 extension that defines
+	// a list of URLs of CA Issuers. If not set, the certificate will be
+	// issued with no CA Issuers servers set. For example, a CA Issuers server
+	// URL could be "http://ca.domain.com/ca.crt".
+	// +optional
+	CaIssuers []string `json:"caIssuers,omitempty"`
```

Comment:
nitpick: I'm not sure about the name since it echoes the name of the struct (`CAIssuer.CAIssuers`). I like the name that's used in the Go stdlib `x509.Certificate` struct. 

Maybe we could use `IssuingCertificateURL` instead (or maybe `IssuingCertificateURLs` since we're dealing with a slice)?

Reply:
I like it, I had similar thoughts when I saw the go struct naming and you've convinced me.  I'll update.